crates/sui-framework/packages/sui-framework/sources/test/test_scenario.move

```diff
@@ -96,9 +100,36 @@ module sui::test_scenario {
 
     /// Begin a new multi-transaction test scenario in a context where `sender` is the tx sender
     public fun begin(sender: address): Scenario {
+        // Randomly seed the transaction context's transaction hash to avoid
+        // object ID collisions in parallel Move tests.
+        //
```

Comment:
I'd remove this comment, since it won't be true long term. 

I would just summarize here that the transaction digest is randomized to better help simulate real execution environments. Not sure we need to say more than that?

I think this block of comments would be a lot better placed in some rust file if possible